Dockerfile.pulumi

```diff
@@ -66,10 +42,7 @@ RUN pip install --no-cache-dir --upgrade pip==21.3.1 wheel==0.37.0 typing_extens
 
 COPY --chown=grapl pulumi pulumi
 
-# We need to parse the following command with `bash`, not `sh`
-SHELL ["/bin/bash", "-c"]
-
-RUN pip install --no-cache-dir -r <(PANTS_IGNORE='[".pex",".cache","venv"]' ./pants dependencies --type=3rdparty pulumi::)
+RUN pip install --no-cache-dir --requirement constraints.txt
```

Comment:
Actually, could you:
1. move this above the COPY pulumi pulumi?
2. remove the `RUN pip install` on 41?

Reply:
Yup